Makefile

```
@@ -145,15 +144,15 @@ kustomize: ## Download kustomize locally if necessary.
 helm-package-operator: helm-cmd helm-test-operator
 	$(HELM) package helm/kfp-operator --version $(VERSION) --app-version $(VERSION) -d dist
 
-helm-package: helm-package-operator
+helm-package: helm-package-operator # Package operator helm-chart
 
-helm-install-operator: helm-package-operator values.yaml
+helm-install-operator: helm-package-operator values.yaml # Install operator
```

```suggestion
helm-install-operator: helm-package-operator values.yaml ## Install operator
```
Needs double `#` to render out

---

same everywhere else that needs to be fixed

---

done in [3cf3259](https://github.com/sky-uk/kfp-operator/pull/486/commits/3cf32592269c585061d07b7d30ddfabe1a3ed601)